core-bundle/contao/templates/twig/backend/template_studio/index.html.twig

```diff
@@ -47,6 +47,22 @@
                     .set('data-contao--template-studio-target', 'tabs'),
                 panels_container_id: 'template-studio--tab-panels',
             } %}
+                {% trans_default_domain "contao_template_studio" %}
+
+                {% block initial_content %}
+                    <div class="explainer">
+                        <h3>{{ 'getting-started.0'|trans }}</h3>
+                        <p>
+                            <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"
```

Comment:
Again, this icon cannot be adapted for the dark mode. Use the `backend_icon()` function instead, please.